src/main/resources/initialize.sql

```diff
@@ -0,0 +1,447 @@
+    "AmountCharged" numeric(15,3) NOT NULL,
+    "IsDeleted" boolean NOT NULL,
+    CONSTRAINT "OnlineDonation_pkey" PRIMARY KEY ("DonationID")
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."OnlineDonation"
+    OWNER to postgres;
+
+
+--7 Table: public.Session
+
+-- DROP TABLE IF EXISTS public."Session";
+
+CREATE TABLE IF NOT EXISTS public."Session"
+(
+    "SessionID" bigint NOT NULL DEFAULT nextval('"Session_SessionID_seq"'::regclass),
+    "Duration" numeric(4,4) NOT NULL,
+    "Date" timestamp without time zone NOT NULL,
+    "IsDeleted" boolean,
+    CONSTRAINT "Session_pkey" PRIMARY KEY ("SessionID")
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."Session"
+    OWNER to postgres;
+
+
+--8 Table: public.Feedback
+
+-- DROP TABLE IF EXISTS public."Feedback";
+
+CREATE TABLE IF NOT EXISTS public."Feedback"
+(
+    "FeedbackID" bigint NOT NULL DEFAULT nextval('"Feedback_FeedbackID_seq"'::regclass),
+    "Comment" text COLLATE pg_catalog."default" NOT NULL,
+    "Stars" smallint NOT NULL,
+    "IsDeleted" boolean NOT NULL,
+    CONSTRAINT "Feedback_pkey" PRIMARY KEY ("FeedbackID")
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."Feedback"
+    OWNER to postgres;
+
+
+--9 Table: public.Topics
+
+-- DROP TABLE IF EXISTS public."Topics";
+
+CREATE TABLE IF NOT EXISTS public."Topics"
+(
+    "TopicsID" integer NOT NULL DEFAULT nextval('"Topics_TopicsID_seq"'::regclass),
+    "TopicsName" text COLLATE pg_catalog."default" NOT NULL,
+    "IsDeleted" boolean NOT NULL,
+    CONSTRAINT "Topics_pkey" PRIMARY KEY ("TopicsID")
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."Topics"
+    OWNER to postgres;
+
+
+--10 Table: public.MT_InterestedIn
+
+-- DROP TABLE IF EXISTS public."MT_InterestedIn";
+
+CREATE TABLE IF NOT EXISTS public."MT_InterestedIn"
+(
+    "MentorID" integer NOT NULL DEFAULT nextval('"MT_InterestedIn_MentorID_seq"'::regclass),
+    "TopicsID" integer NOT NULL DEFAULT nextval('"MT_InterestedIn_TopicsID_seq"'::regclass),
+    CONSTRAINT "MT_InterestedIn_pkey" PRIMARY KEY ("MentorID", "TopicsID"),
+    CONSTRAINT "MT_InterestedIn_MID_fkey" FOREIGN KEY ("MentorID")
+        REFERENCES public."Mentor" ("MentorID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION,
+    CONSTRAINT "MT_InterestedIn_TID_fkey" FOREIGN KEY ("TopicsID")
+        REFERENCES public."Topics" ("TopicsID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."MT_InterestedIn"
+    OWNER to postgres;
+
+
+--11 Table: public.Mentor_Availability
+
+-- DROP TABLE IF EXISTS public."Mentor_Availability";
+
+CREATE TABLE IF NOT EXISTS public."Mentor_Availability"
+(
+    "MentorID" integer NOT NULL DEFAULT nextval('"MentorAvailability_MentorID_seq"'::regclass),
+    "Availability" timestamp without time zone NOT NULL,
+    CONSTRAINT "MentorAvailability_pkey" PRIMARY KEY ("MentorID"),
```

Comment:
please add availability duration. for example: available in 12/12/2012 12:12 for 12 hours